glean-core/ios/sdk_generator.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Glean SDK metrics build script.
+#
+# More about Glean at https://mozilla.github.io/glean
+#
+# This script generates metrics and pings as defined in user-provided files
+# and generates Swift code to be included in the final build.
+# It uses the `glean_parser`.
+# See https://mozilla.github.io/glean_parser/ for details.
+#
+# To use it in a Swift project, follow these steps:
+# 1. Import the `sdk_generator.sh` script into your project.
+# 2. Add your `metrics.yaml` and (optionally) `pings.yaml` to your project.
+# 3. Add a new "Run Script" build step and set the command to `bash $PWD/sdk_generator.sh`
+# 4. Add your definition files (`metrics.yaml`, `pings.yaml`) as Input Files for the "Run Script" step.
+# 5. Run the build once.
+# 6. Add the files in the `Generated` folder to your project.
```

Comment:
So this requires manual input to both set this up in the project *and* use the generated stuff? I wonder if there's a way to automate that..

Reply:
Removed the first manual step (well, people will still want to add it to their project, but they don't need to define it as input files to the script anymore).

They will still need to add the generated stuff so that it gets compiled and there's no way around that short of editing the project file (... and we don't want to go _that_ way, do we?)